tests/e2e/nvidia-container-toolkit_test.go

```diff
@@ -27,69 +29,50 @@ import (
 
 // Integration tests for Docker runtime
 var _ = Describe("docker", Ordered, ContinueOnFailure, func() {
-	var r Runner
-
-	// Install the NVIDIA Container Toolkit
-	BeforeAll(func(ctx context.Context) {
-		r = NewRunner(
-			WithHost(host),
-			WithPort(sshPort),
-			WithSshKey(sshKey),
-			WithSshUser(sshUser),
-		)
-		if installCTK {
-			installer, err := NewToolkitInstaller(
-				WithRunner(r),
-				WithImage(image),
-				WithTemplate(dockerInstallTemplate),
```

Comment:
Does this mean we're no longer only doing this for Docker tests?

Reply:
I moved this to the `BeforeSuite`, as for now this E2E suite is focused on docker, once we extend to other runtimes we can move to a different structure

Reply:
What motivated doing this move now?

Reply:
To centralize all the prep work during BeforeSuite. the installer part is prep work, so it makes sense to host it on the BeforeSuite

Reply:
Since we don't use the runner outside of these tests cases at all, let's rather move the initialization back here. As I pointed out, this also makes it simpler to extend our tests to other runtimes since the docker configuration is localized.